config-mcp.properties

```diff
@@ -0,0 +1,28 @@
+# YaCy Grid Peer configuration
```

Comment:
copy this configuration file to data/mcp-8100/conf/config.properties
using cp command